src/schema/__tests__/partner_show_events.test.ts

```diff
@@ -0,0 +1,154 @@
+import { runQuery } from "test/utils"
+import gql from "lib/gql"
+import moment from "moment"
+import { ResolverContext } from "types/graphql"
+
+// This time-bomb spec will fail starting at some defined point in the future,
+// indicating that we should consider reverting this behavior of sending
+// different raw date representations to different versions of Eigen
+// (in the hope that usage of older versions has dropped below some minimum threshold).
+
+it("is not yet time to rethink this UA-sniffing behavior for resolving dates", () => {
+  const today = moment()
+  const deadline = moment("2020-10-01") // about 18 months after adding this behavior
```

Comment:
I picked this number based on the number that came up in the Zoom chat, but open to changes here.

Reply:
I like the idea of an automated test here, but I think the most actionable thing to do is start to work with PM/TLs to prioritize this work and get it on Jira. I encourage keeping this test around and starting the prioritization discussion.

Reply:
Agreed 👍

Reply:
This might be the first real instance of the kind of time-bomb test I imagined, so thanks. 😍 

I would expect to see a more specific assertion here, like `expect(data).toEqual("... +00:00")` which would fail after the deadline. It looks like that would require piggy-backing on an existing test or lots of repeat stubbing, so this is fine too.

Reply:
💣 💥 https://artsy.slack.com/archives/C02BC3HEJ/p1601567857100300

Reply:
:ceiling-cat: Did something explode?

Reply:
Ah, I see it now 😄

Reply:
Yes, just noting that the bomb went off as planned 😄 

(And hello, friend!)

Reply:
👋  -so cool to see this worked!

Reply:
👋 Best date. When do we meet next time?

Reply:
Hey @dleve123 👋 . Yeah this spec was just was disruptive as we... wanted?.. it to be? Lol

I'll go write another time bomb so we can meet again.